frontend/src/routes/candidate/register/PasswordSetPage.svelte

```diff
@@ -0,0 +1,112 @@
+<script lang="ts">
+  import {_} from 'svelte-i18n';
+  import {page} from '$app/stores';
+  import Footer from '$lib/components/footer/Footer.svelte';
+  import PasswordValidator from './PasswordValidator.svelte';
+  import {validatePassword} from './passwordValidation';
+
+  export let userName = 'Barnabas'; /** TODO: Ensure name is set correctly */
+
+  let password1 = '';
+  let password2 = '';
+  let validPassword = false;
+  let passwordsMatch = true;
+
+  $: disableSetButton = validPassword && password2.length > 0;
+
+  const onSetButtonPressed = async () => {
+    if (password1 != password2) {
+      passwordsMatch = false; // Display error message
+      return;
+    }
+
+    passwordsMatch = true;
+
+    // Additional check before backend validation
+    if (!validatePassword(password1, userName)) {
+      return;
+    }
+  };
+
+  /** TODO: Backend stuff */
+</script>
+
+<!--
+@component
+Page where candidates can set their password when logging to the app for the first time.
+
+### Properties
+
+- userName:
+  - The component will greet the user with the given name.
+  - Default value is Barnabas, but this will be fixed later.
+
+### Usage
+  ```tsx
+  <PasswordSetPage userName={"Barnabas"}/>
+    ```
+-->
+
+<div class="flex w-full flex-grow flex-col items-center bg-base-300">
+  <main class="flex-grow">
+    <div class="flex max-w-xl flex-col items-center p-lg pl-safelgl pr-safelgr">
+      <div class="flex flex-col flex-nowrap items-center">
+        <hgroup class="py-lg">
+          <p class="text-2xl font-bold text-primary">{$page.data.appLabels.appTitle}</p>
+          <h1 class="text-3xl font-normal">{$page.data.election.name}</h1>
+          <h1 class="my-24 text-2xl font-normal">
+            {$_('candidateApp.setPassword.greeting')}{userName}!
```

Comment:
I think that this would be better if done in the way as it was done in [the log out modal](https://github.com/OpenVAA/voting-advice-application/blob/6d591f55081ff6605af9af34ede75fc8e0e0dc2e/frontend/src/lib/components/navigation/Navigation.svelte#L62). Or at least I would move the trailing space from en.json to here because it is probably easy to forgot when creating new translations.